src/main/java/apoc/ApocConfiguration.java

```diff
@@ -28,6 +30,10 @@
     public static void initialize(GraphDatabaseAPI db) {
         Static.clear();
         Map<String, String> params = db.getDependencyResolver().resolveDependency(Config.class).getRaw();
+        Map<String, String> configValues = db.getDependencyResolver().resolveDependency(Config.class)
```

Comment:
What's the difference?

Reply:
We merged `params` and `configValues` because all non-neo4j configuration such as apoc.* is not included into configValues; we don't get just a singular configuration but the whole one, some other parameter can be useful for some other feature; but if you want we could get them one by one by the need

Reply:
As there might be some security critical things, I'd prefer to only extract the ones we need.